docs/src/VariableList.md

```diff
@@ -102,6 +102,50 @@ conv_q_liq_to_q_rai_accr = tendency to q_rai due to accretion from q_liq [1/s]
 conv_q_rai_to_q_vap      = tendency to q_vap due to evaporation from q_rai [1/s]
 ```
 
+### 2.7 Diagnostics
+Here are suggested names in the output. Format: short name (code name) = long name
+```
+rho(ρ) = density
+u(u) = x_velocity
+v(v) = y_velocity
+w(w) = w_velocity
+qt(q_tot) = total specific humidity
+et(e_tot) = total specific energy
+qv(q_vap) = water vapor specific humidity
+ql(q_liq) = liquid water specific humidity
+ei(e_int) = specific internal energy
+thd(θ_dry) = potential temperature
+thl(θ_liq_ice) = liquid-ice potential temperature
+thv(θ_vir) = virtual potential temperature
+hm(h_moi) = specific enthalpy
+ht(h_tot) = total specific enthalpy
+
+var_u(u′u′) = variance of x-velocity
+var_v(v′v′) = variance of y-velocity
+var_w(w'w') = variance of z-velocity
+TKE(TKE) = turbulence kinetic energy
+var_qt(q_tot'q_tot') = variance of total specific humidity
+var_ei(e_int'e_int') = variance of specific internal energy
+var_thl(θ_liq_ice'θ_liq_ice') = variance of liquid-ice potential temperature
+cov_qt_thl(q_tot'θ_liq_ice') = covariance of total specific humidity and liquid-ice potential temperature
+cov_qt_ei(q_tot'e_int') = covariance of total specific humidity and specific internal energy
+wskew(w′w′w′) = skewness of w-velocity
```

Comment:
I'd only call it skewness if you divide by variance^(3/2). Otherwise, perhaps `w3` or something like it is better?